README.md

```diff
@@ -18,6 +18,8 @@ The Stripe Flutter SDK allows you to build delightful payment experiences in you
 
 **Apple Pay**: We provide a [seamless integration with Apple Pay](https://stripe.com/docs/apple-pay).
 
+**Google Pay**: By integrating the [Pay plugin](https://pub.dev/packages/pay) we provide a seamless integration with Google pay. All you need to do is add your stripe publishable key to the payment profile. See our example for more information. 
```

Comment:
I'll add a short standalone heading regarding the pay plugin to the readme tonight :)

Reply:
done